applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -540,9 +540,32 @@ Syntax
   * ``0`` - Get
   * ``1`` - Set
 
-For a complete list of the supported SET ``<name>`` accepted parameters, see the `SETSOCKETOPT Service Spec Reference`_.
+* The ``<name>`` parameter can accept one of the following values:
```

Comment:
There is already a list of supported socket options [here](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_modem/doc/sockets.html#socket-options).

Reply:
I was not aware of that list, I will refer it for descriptions. However, SLM does not implement all of the options, so I would like to keep the list in here as well.